packages/api/src/models/Predicate.ts

```diff
@@ -22,7 +22,7 @@ export interface PredicateMember {
 
 @Entity('predicates')
 class Predicate extends Base {
-  @Column()
+  @Column({ unique: true })
```

Comment:
This changes not refleted on DB
We need create a new index to this column using a migration

Check more infos [here](https://typeorm.io/migrations) and find by `createIndex()`